packages/medusa/src/services/price-list.ts

```diff
@@ -359,6 +356,109 @@ class PriceListService extends BaseService {
     })
   }
 
+  async listVariants(
+    priceListId: string,
+    selector: FilterableProductVariantProps = {},
+    config: FindConfig<ProductVariant> = {
+      relations: [],
+      skip: 0,
+      take: 20,
+    },
+    requiredRelation = false
```

Comment:
comment: Same as other comment. I believe we should improve the readability of this prop

Reply:
Same